src/components/invest/investslug/components/AuthorSection.tsx

```diff
@@ -0,0 +1,33 @@
+'use client'
+import { cn } from '@/lib/utils'
+import { AvatarImage } from '@/components/shared/AvatarImage'
+
+interface AuthorSectionProps {
+  author: string
+  imageURL: string
+  tags: string[]
+}
+
+export const AuthorSection: React.FC<AuthorSectionProps> = ({ author, imageURL, tags }) => {
+  return (
+    <section className='w-full py-2 md:py-8 flex flex-row gap-4 md:gap-10 justify-between items-center'>
```

Comment:
I'm trying to use `flex flex-col` with `gap` instead of adding paddings/margins to make space between two different components. We are good for now, but just to keep that in mind